gdal/doc/source/drivers/raster/jp2grok.rst

```diff
@@ -0,0 +1,357 @@
+.. _raster.jp2grok:
+
+================================================================================
+JP2Grok -- JPEG2000 driver based on Grok library
+================================================================================
+
+.. shortname:: JP2Grok
+
+.. build_dependencies:: grok >= 8.0
+
+
+This driver is an implementation of a JPEG2000 reader/writer based on
+the Grok library.
+
+The driver uses the VSI Virtual File API, so it can read JPEG2000
+compressed NITF files.
+
+XMP metadata can be extracted from JPEG2000 files, and will be stored as
+XML raw content in the xml:XMP metadata domain.
+
+The driver supports writing georeferencing information as GeoJP2 and
+GMLJP2 boxes.
+
+The driver supports creating files with
+transparency, arbitrary band count, and adding/reading metadata. Update
+of georeferencing or metadata of existing file is also supported.
+Optional intellectual property metadata can be read/written in the
+xml:IPR box.
+
+Driver capabilities
+-------------------
+
+.. supports_createcopy::
+
+.. supports_georeferencing::
+
+.. supports_virtualio::
+
+Georeferencing
+--------------
+
+Georeferencing information can come from different sources : internal
+(GeoJP2 or GMLJP2 boxes), worldfile .j2w/.wld sidecar files, or PAM
+(Persistent Auxiliary metadata) .aux.xml sidecar files. By default,
+information is fetched in following order (first listed is the most
+prioritary): PAM, GeoJP2, GMLJP2, WORLDFILE.
+
+Starting with GDAL 2.2, the allowed sources and their priority order can
```

Comment:
I think the mention of GDAL 2.2 can be omitted since this is a new driver in a later version.